src/pages/Plant.js

```diff
@@ -1,17 +1,25 @@
+
 import React from "react";
 const Plant = (props) => {
 
-const { plants } = props
+    const plant = [
+        {name: "Coctus",
+         img:"https://canary.contestimg.wish.com/api/webimage/5c7fa4fefc2f9d48b8020088-large.jpg?cache_buster=8f0d33c27ee1d5acbebf084b43b6a520", alt:"plant",
+        frequency: 2, 
+        sunlight: "direct", 
+        temperature: { min: 45, max: 80},
+        description: "This is my favorite plant"}
+      ]
 
      return <div className='individualPlant'>
     
-        <article key={plant._id}>
-          <img src={plant.img}/>
-          <h1>{plant.name}</h1>
-          <h4>Fruequency: {plant.frequency}</h4>
-          <h4>Sunlight: {plant.sunligh}</h4>
-          <h4>Temprature: {plant.tempreture}</h4>
-          <h4>Description: {plant.description}</h4>
+        <article key={props.plant._id}>
```

Comment:
This is the only thing i'm not sure about. I think here it's props.plant.id as _id will use the obj id, but I could be wrong

Reply:
You only need to include a key attribute if you're creating multiple of the same component, like through array.map()

This key attribute can be omitted.